server/config/config.go

```diff
@@ -868,12 +869,8 @@ func (c *ScheduleConfig) adjust(meta *configMetaData, reloading bool) error {
 		c.StoreLimit = make(map[uint64]StoreLimitConfig)
 	}
 
-	if !meta.IsDefined("hot-regions-write-interval") {
```

Comment:
why remove this?

Reply:
just want to keep consistency with other intervals, shoud we add this line back?
```go
	adjustDuration(&c.SplitMergeInterval, defaultSplitMergeInterval)
	adjustDuration(&c.PatrolRegionInterval, defaultPatrolRegionInterval)
	adjustDuration(&c.MaxStoreDownTime, defaultMaxStoreDownTime)
	adjustDuration(&c.HotRegionsWriteInterval, defaultHotRegionsWriteInterval)
```

Reply:
I think we better keep this part.